src/CoreFoundation.zig

```diff
@@ -75,6 +226,31 @@ pub extern "c" fn CFRelease(*anyopaque) void;
 
 test {
     _ = testing.refAllDecls(@This());
+    _ = testing.refAllDecls(ZigAllocatorCallbacks);
+    _ = testing.refAllDecls(CFAllocator);
+    _ = testing.refAllDecls(CFAllocator.CFAllocatorContext);
     _ = testing.refAllDecls(CFData);
+    _ = testing.refAllDecls(CFDate);
     _ = testing.refAllDecls(CFString);
 }
+
+test "Basic Date Tests" {
+    const allocator = testing.allocator;
+    const cfallocator = try CFAllocator.createFromZigAllocator(&allocator, CFAllocator.kCFAllocatorUseContext);
+    defer CFRelease(cfallocator);
+
+    const early_date = CFDate.CFDateCreate(cfallocator, 0).?;
```

Comment:
Building on the previous comment, I'd rather we tested the Zig API we want to expose. For testing `extern`s I believe just having the project compile should be enough. Lemme know what you think!

Reply:
yeah - I think that's a fair point - this was mainly to test the zig allocator wrapper worked. Although I just realised there's some alignment stuff (due to the fact we have to store the size ourselves :( ) I need to ensure is sorted first. So will do that as well. Will remove this test.

Reply:
Oh, don't worry too much about not everything working out from a get-go - the way I see this project is that it's very much experimental and we are more than fine to include wrappers which are not 100% yet.